spec/compiler/parser/parser_spec.cr

```diff
@@ -1787,6 +1787,14 @@ module Crystal
       end
     ), ClassDef.new("Foo".path, Def.new("bar", body: Call.new(nil, "print", Cast.new(Var.new("self"), "Foo".path))))
 
+    it_parses %(
+      class ÁrvíztűrőTükörfúrógép
+        def árvíztűrő_tükörfúrógép
+          print as ÁrvíztűrőTükörfúrógép
+        end
```

Comment:
The `as` syntax is obsolete.

Also, what's the point of the method def? Wouldn't the class definition alone be sufficient?
```suggestion
```